testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -97,15 +98,18 @@ def test_convert_aa_1to3(self, resname1, strings):
 
     @pytest.mark.parametrize('x', (
             'XYZXYZ',
-            '£'
+            'Â£'
```

Comment:
The Windows powershell vim is ninja-converting this on me. Is this because the header encoding is not processed properly on Windows?